### PR TITLE
[4.0] API language strings

### PR DESCRIPTION
**Pull request by brianteeman**
Ensure that the new json api code uses language strings and not hard coded text

code review

### CODE REVIEW

File: administrator/language/en-GB/en-GB.lib_joomla.ini [hunk 1]
```diff
@@ -24,29 +24,38 @@ JLIB_APPLICATION_ERROR_CHECKOUT_FAILED="Check-out failed with the following erro
...
 JLIB_APPLICATION_ERROR_COMPONENT_NOT_LOADING="Error loading component: %1$s, %2$s"
+JLIB_APPLICATION_ERROR_CONTENT_TYPE_MISSING="Content type missing."
 JLIB_APPLICATION_ERROR_CONTROLLER_GET_NAME="JController: :getName() : Can't get or parse class name."
 JLIB_APPLICATION_ERROR_CREATE_RECORD_NOT_PERMITTED="Create record not permitted."
+JLIB_APPLICATION_ERROR_DELETE="There was an error deleting the item."
 JLIB_APPLICATION_ERROR_DELETE_NOT_PERMITTED="Delete not permitted."
 JLIB_APPLICATION_ERROR_EDITSTATE_NOT_PERMITTED="Edit state is not permitted."
 JLIB_APPLICATION_ERROR_EDIT_ITEM_NOT_PERMITTED="Edit is not permitted."
 JLIB_APPLICATION_ERROR_EDIT_NOT_PERMITTED="Edit not permitted."
+JLIB_APPLICATION_ERROR_DISPLAY_EMAIL_FORM="Display email form disabled."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.lib_joomla.ini]: Move after line 31.

File: language/en-GB/en-GB.lib_joomla.ini [hunk 2]
```diff
@@ -24,28 +24,38 @@ JLIB_APPLICATION_ERROR_CHECKOUT_FAILED="Check-out failed with the following erro
...
 JLIB_APPLICATION_ERROR_COMPONENT_NOT_LOADING="Error loading component: %1$s, %2$s"
+JLIB_APPLICATION_ERROR_CONTENT_TYPE_MISSING="Content type missing."
 JLIB_APPLICATION_ERROR_CONTROLLER_GET_NAME="JController: :getName() : Can't get or parse class name."
 JLIB_APPLICATION_ERROR_CREATE_RECORD_NOT_PERMITTED="Create record not permitted."
+JLIB_APPLICATION_ERROR_DELETE="There was an error deleting the item."
 JLIB_APPLICATION_ERROR_DELETE_NOT_PERMITTED="Delete not permitted."
 JLIB_APPLICATION_ERROR_EDITSTATE_NOT_PERMITTED="Edit state is not permitted."
 JLIB_APPLICATION_ERROR_EDIT_ITEM_NOT_PERMITTED="Edit is not permitted."
 JLIB_APPLICATION_ERROR_EDIT_NOT_PERMITTED="Edit not permitted."
+JLIB_APPLICATION_ERROR_DISPLAY_EMAIL_FORM="Display email form disabled."
```
Comment (Quy) [inline language/en-GB/en-GB.lib_joomla.ini]: Sort.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1b3641e5d7ea26089ae852a1371d8de99f05c28d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26393">issues.joomla.org/tracker/joomla-cms/26393</a>.</sub>

**Comment by wilsonge** [comment]
Sorry are you able to fix conflicts - I finally found why the API tests were failing and fixed it - but broke this with the file rename

**Comment by brianteeman** [comment]
@wilsonge conflicts fixed - just waiting to see if the tests complete successfully

**Comment by brianteeman** [comment]
all tests passed. as this last change was just to resolve merge conflicts it is probably rtc

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**